File: lib/arrays.rb
genres = %w[rock pop jazz indie]

# C-reate
genres << 'house'
genres.push('classical')

# R-ead
puts genres[1]

# U-pdate
genres[0] = 'blues'

# D-elete
genres.delete_at(1) # deletes 'pop'

File: lib/arrays.js
const genres = ['rock', 'pop', 'jazz', 'indie'];
console.log(genres);

// C-reate
genres.push('classical');
console.log(genres);

// R-ead
console.log(genres[1]);

// U-pdate
genres[0] = 'blues';
console.log(genres);

// D-elete
genres.splice(1, 1);
console.log(genres);
